ui/src/pages/publications/[id]/versions/[versionId].tsx

```diff
@@ -555,6 +555,44 @@ const Publication: Types.NextPage<Props> = (props): React.ReactElement => {
                         />
                     )}
 
+                    {
+                        // Show details of publication version Peer Review is linked to.
+                        publication?.type === 'PEER_REVIEW' &&
```

Comment:
This is difficult to read. Reckon it could be better moving some of these checks into separate variables, then generate the component in a separate function. Variables names are all guesses at what is actually being checked here, but hopefully you get the gist. 

```
const generateAlertComponent = () => {
    let isPeerReview = publication?.type === 'PEER_REVIEW' && linkedTo.length === 1
    let hasNewerVersion = linkedTo[0].parentVersionNumber && linkedTo[0].parentVersionId

    if (!isPeerReview) {
        return null; // early return
    }
    
    if (hasNewerVersion) {
         return (
            ...
            'This peer review covers content that has been replaced by a newer version, and is therefore potentially outdated.'
           ...
         )
    } else {
        ...
        This is a peer review of the latest version of the following publication:{' '}
        ...
    }
};

let alertComponent = generateAlertComponent();
```

Last bit then is to add the alertComponent into the html

Reply:
That's a good idea; I ended up grouping up the different types of alerts into one array and spaced out the logic a bit with more comments - hopefully it's a bit better now

Reply:
Very nice, I like it, good job :+1: